packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.types.ts

```diff
@@ -0,0 +1,15 @@
+import { Align, Justify, TooltipProps } from '@leafygreen-ui/tooltip';
+
+export { Align, Justify };
+
+type ModifiedTooltipProps = Omit<
+  TooltipProps,
+  'onClick' | 'trigger' | 'triggerEvent' | 'refEl' | 'children'
+>;
+
+export interface InfoSprinkleProps extends ModifiedTooltipProps {
+  /**
+   * String rendered inside the tooltip
+   */
+  children: string;
```

Comment:
Why are we restricting `children` to be a string? Can we add the reasoning to the docs

Reply:
Based on the guidelines, only text is allowed but I'm actually not sure if there can be multiple lines/paragraphs. @alvenw Can there be multiple lines/paragraphs? 

If that's the case then i can switch this to a reactnode.

Reply:
I think I agree that this should just be a string (otherwise we open it up to rendering Links)

Reply:
Yep the tooltip content should be short and concise. Anything beyond a couple lines of text would warrant having it be surfaced elsewhere in the UI